Library/Formula/zeromq.rb

```diff
@@ -28,6 +28,12 @@ class Zeromq < Formula
   depends_on "libpgm" => :optional
   depends_on "libsodium" => :optional
 
+  if not build.head?
+    # enable --without-libsodium on libzmq < 4.1
+    # zeromq/zeromq4-x#105
+    patch :DATA
```

Comment:
Oh of course! In that case I wouldn't worry too much about it then. A gist would look nicer, but don't fret too much if it's a pain. We don't really want to force an ` autogen ` as that'll enforce an autotools build-time dep on ` -fs ` users.